projects/plugins/jetpack/_inc/lib/class-jetpack-ai-helper.php

```diff
@@ -154,14 +154,29 @@ public static function get_gpt_completion( $content, $post_id ) {
 				\require_lib( 'openai' );
 			}
 
-			$result = ( new OpenAI( 'openai', array( 'post_id' => $post_id ) ) )->request_gpt_completion( $content );
+			// Set the content for chatGPT endpoint
+			$data = array(
+				array(
+					'role'    => 'user',
+					'content' => $content,
+				),
+			);
+
+			$result         = ( new OpenAI( 'openai', array( 'post_id' => $post_id ) ) )->request_chat_completion(
+				$data,
+				$max_tokens = 140
```

Comment:
Let's have this limit in the openai library so we can tweak it

Reply:
Great idea!